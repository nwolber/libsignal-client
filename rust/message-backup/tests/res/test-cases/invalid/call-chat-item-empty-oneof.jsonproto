[
  {
    "version": "1",
    "backupTimeMs": "1705692409729"
  },
  {
    "recipient": {
      "id": "1",
      "contact": {
        "aci": "IiMfftM7TXC2Brinq8Wbvw==",
        "registered": "REGISTERED"
      }
    }
  },
  {
    "recipient": {
      "id": "2",
      "self": {}
    }
  },
  {
    "chat": {
      "id": "1",
      "recipientId": "2"
    }
  },
  {
    "chatItem": {
      "chatId": "1",
      "authorId": "2",
      "directionless": {},
      "updateMessage": {
        "callingMessage": {}
      }
    }
  }
]