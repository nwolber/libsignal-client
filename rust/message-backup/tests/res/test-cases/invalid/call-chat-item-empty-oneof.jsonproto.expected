chat frame ChatId(1) error: chat item: CallChatUpdate.chatUpdate is a oneof but is empty